java/client/src/main/java/glide/api/models/commands/scan/ClusterScanCursor.java

```diff
@@ -86,7 +86,17 @@ public void releaseCursorHandle() {
             };
 
     /** Creates an empty cursor to be used in the initial {@link GenericClusterCommands#scan} call. */
-    static ClusterScanCursor initalCursor() {
+    static ClusterScanCursor initialCursor() {
```

Comment:
This is unfortunately a breaking change so we technically should keep initalCursor() and mark it deprecated.